src/blocks/card.css

```diff
@@ -0,0 +1,83 @@
+.card {
+  background: #fff;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  width: 200px;
+  padding: 16px;
+}
+
+.card__screen {
+  height: 134px;
+  width: 160px;
+  object-fit: cover;
+  margin: 16px 3px 36px;
+}
+
+.card__title {
+  display: flex;
+  font-family: "Roboto", sans-serif;
+  justify-content: center;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin-bottom: 4px;
+}
+
+.card__code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  padding-bottom: 16px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  padding-bottom: 16px;
+}
+
+.card__title-price {
+  font-weight: 400;
+  font-size: 12px;
+  color: #616070;
+  padding-right: 3px;
+}
+
+.card__number-price {
+  align-self: flex-end;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.card__button {
+  padding: 0;
+  border: none;
+  width: 100%;
+  border-radius: 5px;
+}
+
+.card__link-button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background: #00acdc;
+  border-radius: 5px;
+  font-family: "Roboto", sans-serif;
+  padding: 12px 0;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-transform: uppercase;
+  color: #fff;
+  text-decoration: none;
+}
+
+.card__link-button:hover {
+  padding: 11px 0;
```

Comment:
You dont have to change padding on hover, add initial transparent border and use box-sizing: border-box to every element (* selector)